### ISSUE TITLE
Question about meta-validation during meta-train

### ISSUE THREAD

**Issue by vickichen97** [yaoyao-liu/meta-transfer-learning#14]
Whether the meta-validation data classes are consistent with the meta-train data classes in one task? For meta-validation during meta-train, the meta_lr is 0 which means ss_weights  and fc_weights  will not be updated. Whether the updates in fast_fc_weights will not influence fc_weights when the meta_lr is 0 ?
Looking forward to your reply,thanks!

**Comment by yaoyao-liu** [comment]
Answer to Q1: Meta-train, meta-val, and meta-test set share no common class, i.e., the class occurs in meta-train won't occur in meta-val and meta-test.

Answer to Q2: During base-learning, the fc weights are detached from the parameter list so the updates won't be stored.
https://github.com/yaoyao-liu/meta-transfer-learning/blob/2fd01a254d580909e5c1c0100347ab71216aad62/pytorch/models/mtl.py#L98

**Comment by vickichen97** [comment]
Thank you very much！
